jaxtronomy/LensModel/Solver/lens_equation_solver.py

```diff
@@ -0,0 +1,773 @@
+            x_mins = x_mins[mag >= magnification_limit]
+            y_mins = y_mins[mag >= magnification_limit]
+        return x_mins, y_mins
+
+    def findBrightImage(
+        self,
+        sourcePos_x,
+        sourcePos_y,
+        kwargs_lens,
+        numImages=4,
+        min_distance=0.01,
+        search_window=5,
+        precision_limit=10 ** (-10),
+        num_iter_max=100,
+        arrival_time_sort=True,
+        x_center=0,
+        y_center=0,
+        num_random=0,
+        magnification_limit=None,
+        initial_guess_cut=False,
+        verbose=False,
+    ):
+        """
+
+        :param sourcePos_x: source position in units of angle
+        :param sourcePos_y: source position in units of angle
+        :param kwargs_lens: lens model parameters as keyword arguments
+        :param min_distance: minimum separation to consider for two images in units of angle
+        :param search_window: window size to be considered by the solver. Will not find image position outside this window
+        :param precision_limit: required precision in the lens equation solver (in units of angle in the source plane).
+        :param num_iter_max: maximum iteration of lens-source mapping conducted by solver to match the required precision
+        :param arrival_time_sort: bool, if True, sorts image position in arrival time (first arrival photon first listed)
+        :param initial_guess_cut: bool, if True, cuts initial local minima selected by the grid search based on distance criteria from the source position
+        :param verbose: bool, if True, prints some useful information for the user
+        :param x_center: float, center of the window to search for point sources
+        :param y_center: float, center of the window to search for point sources
+        :param num_random: int, number of random positions within the search window to be added to be starting
+         positions for the gradient decent solver
+        :param non_linear: bool, if True applies a non-linear solver not dependent on Hessian computation
+        :param magnification_limit: None or float, if set will only return image positions that have an
+         abs(magnification) larger than this number
+        :returns: (exact) angular position of (multiple) images ra_pos, dec_pos in units of angle
+        """
+
+        x_mins, y_mins = self.image_position_from_source(
+            sourcePos_x,
+            sourcePos_y,
+            kwargs_lens,
+            min_distance=min_distance,
+            search_window=search_window,
+            precision_limit=precision_limit,
+            num_iter_max=num_iter_max,
+            arrival_time_sort=arrival_time_sort,
+            initial_guess_cut=initial_guess_cut,
+            verbose=verbose,
+            x_center=x_center,
+            y_center=y_center,
+            num_random=num_random,
+            magnification_limit=magnification_limit,
+        )
+        mag_list = []
+        for i in range(len(x_mins)):
```

Comment:
you actually don't need a for-loop here. Instead:
mag_list = np.abs(self.lensModel.magnification(x_mins, y_mins, kwargs_lens))
I corrected it just in lenstronomy as well

Reply:
Done